### PR TITLE
added list box

### PR DESCRIPTION
**Pull request by jameskokoska**
image

### CODE REVIEW

**Review by natapokie — APPROVED** [review]
looks good!! and i was wondering if the list box is only for one or two words?

**Comment by jameskokoska** [comment]
> looks good!! and i was wondering if the list box is only for one or two words? if there are more words, i think wrapping the text would be good too! image

Text does wrap actually. CSS can be dumb sometimes. The way CSS works is that a very long word won't wrap and that's expected behavior to go off screen. It would be ideal to ellipsis or clip, but a lot of times that isn't standard practice in CSS
image

**Comment by natapokie** [comment]
> Text does wrap actually. CSS can be dumb sometimes. The way CSS works is that a very long word won't wrap and that's expected behavior to go off screen. It would be ideal to ellipsis or clip, but a lot of times that isn't standard practice in CSS image

ohh i see! good to know!

**Review by Cali0707 — APPROVED** [review]
Looks good!

**Comment by Cali0707** [comment]
@jameskokoska can you fix merge conflicts?

File: client/src/App.css [hunk 1]
```diff
@@ -15,7 +15,7 @@
   --text-link: #2680c1;
   --text-dark: #000000;
   --text-light: #f3f3f3;
-  --text-muted: #c2c2c2;
+  --text-muted: #696969;
```
[the remark this answers is not in the document]
Comment (jameskokoska) [reply client/src/App.css:18]: consider it an Easter egg

**Pull request merged**